js/client-teambuilder.js

```diff
@@ -980,12 +980,16 @@
 				if (this.curSetList.length && !this.curSetList[this.curSetList.length - 1].species) {
 					this.curSetList.splice(this.curSetList.length - 1, 1);
 				}
+
+				var teamIsEmpty = this.curSetList.length > 0;
```

Comment:
Uh, this means the team is not empty...

Anyway, `!this.curSetList.length` is short enough that I think I'd actively object to having a temporary variable for it here.